### PR TITLE
Adding comments button

### PR DESCRIPTION
**Pull request by lqueipom**

### CODE REVIEW

**Comment by LukeGMTravis** [comment]
Starting review

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,84 +14,80 @@
...
+  private final List<Object> jsonVersion = new ArrayList<>();
 
-  private List<String> messages;
-
-    @Override
-    public void init() {
-      messages = new ArrayList<String>();
+  @Override 
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    int amount =  Integer.parseInt(request.getParameter("amount"));
```
Comment (LukeGMTravis) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: Nit: extra space

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -46,6 +46,21 @@ function getJSONString() {
   });
 }
 
+function loadingMyComments() {
+    // Stats is an object, not a string, so we have to
+    // reference its fields to create HTML content.
+    document.getElementById('log').innerHTML = "";
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:52]: Be consistent with single vs double quotes in JS

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -46,6 +46,21 @@ function getJSONString() {
...
+    let elem = Integer.parseInt(document.getElementById("amount").value);
+    if (elem != 0) {
+      fetch('/data').then(response => response.json()).then((jsonVersion) => {
+        const oneComment = document.getElementById('log');
+        for (let i=0; i<elem; i++) {
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:57]: Nit: add spaces around operators

Comment (lqueipom) [reply portfolio/src/main/webapp/script.js]: Just resolved all nits and fixed some functionality issues that I had before, thank you for the comments let me know if anything else needs work!

**Pull request merged**